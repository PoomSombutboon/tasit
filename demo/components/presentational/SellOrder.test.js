import React from "react";
import { shallow } from "enzyme";
import SellOrder from "./SellOrder";

describe("SellOrder", () => {
  it("renders the component", async () => {
    expect(
      shallow(<SellOrder land={{}} onClaim={() => {}} />)
    ).toMatchSnapshot();
  });
});
