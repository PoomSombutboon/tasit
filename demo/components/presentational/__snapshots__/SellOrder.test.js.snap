// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SellOrder renders the component 1`] = `
<Component
  style={
    Object {
      "width": 450,
    }
  }
>
  <EstateForSale
    estate={
      Object {
        "id": -1,
        "img": "https://decentraland.org/images/logo-65f7b27caf.png",
        "name": "Not found",
      }
    }
  />
  <Component>
    0
     MANA (~$
    0
    )
  </Component>
</Component>
`;
